frontend/src/metabase/models/components/ModelDetailPage/ModelInfoSidePanel/ModelRelationships.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+
+import Icon from "metabase/components/Icon";
+
+import type Question from "metabase-lib/Question";
+import type Table from "metabase-lib/metadata/Table";
+
+import { ModelInfoTitle, ModelInfoSection } from "./ModelInfoSidePanel.styled";
+import { List, ListItemLink, ListItemName } from "./ModelRelationships.styled";
+
+interface Props {
+  model: Question;
+  mainTable?: Table | null;
+}
+
+function ModelRelationships({ model, mainTable }: Props) {
+  const relatedTables = useMemo(() => {
+    const tablesMainTablePointsTo = model.table()?.foreignTables() || [];
+    const tablesPointingToMainTable = mainTable?.connectedTables() || [];
+    return _.uniq(
+      [...tablesMainTablePointsTo, ...tablesPointingToMainTable],
+      table => table.id,
+    );
+  }, [model, mainTable]);
+
+  if (relatedTables.length <= 0) {
```

Comment:
interestingly this is the only place in our code that uses `.length <= 0` 😮 .

I think we should follow the train and use `.length === 0` here, since `<= 0` implies it can be negative when length can only be either 0 or positive numbers.